src/index.js

```diff
@@ -84,9 +84,9 @@ function createLoader() {
 function initializePal(loader) {
   let type;
 
-  const isElectronRenderer = isNodeLike && process.type === 'renderer';
+  var isRenderer = isNodeLike && (process.type === 'renderer' || process.versions['node-webkit']);
```

Comment:
One more questions, is it: 
`versions['node-webkit']`
or
`versions['nw']`
?

Asking, cause you used both in the two of your commits.